### DISCUSSION TITLE
I can't get Instagram messages

### DISCUSSION THREAD

**Discussion by qqenglishbr** [chatwoot/chatwoot#5902] [Self hosted]
Hello dear all,
I have a new self hosted chatwoot. I follow all setup documentations and it's working well with email, whatsapp and facebook, but I'm not receiving messages that come from Instragram, just facebook messages.

in the webhook callback I tried mydomain/bot and mydomain/webhooks/instagram

I will be grateful if someone gives me some advice

**Comment by engmsaleh** [comment]
I think the problem is that you didn't get the permissions from Facebook for permissions required
